python/setup.py

```diff
@@ -39,7 +39,7 @@ def has_ext_modules(self):
 
 setup(name='weld',
       version='0.0.1',
-      packages=['weld'],
+      packages=['weld', 'grizzly'],
```

Comment:
Installing multiple top level packages might be a little bit counterintuitive for a lot of Python programmers. I would encourage you to set up a single top level namespace entry point (like `import weld.grizzly as gr`)